ambari-server/src/main/java/org/apache/ambari/server/orm/entities/HostStateEntity.java

```diff
@@ -137,7 +139,7 @@ public boolean equals(Object o) {
 
     HostStateEntity that = (HostStateEntity) o;
 
-    if (hostId != null ? hostId != that.hostId : that.hostId != null) return false;
+    if (hostId != null ? !Objects.equals(hostId, that.hostId) : that.hostId != null) return false;
```

Comment:
This can be simplified to:
```
if (!Objects.equals(hostId, that.hostId)){
   retur false;
}
```

Reply:
It's done. Need I change all the codes left in equal() method using Objects.equals?

Reply:
Yes, that would be great

Reply:
It's done. I have also changed hashCode() as well, using Objects.hash(). Please have a look.